src/reaper_osara.cpp

```diff
@@ -770,14 +795,12 @@ void postGoToTrack(int command, MediaTrack* track) {
 	if (trackNum > 0) { // Not master
 		int folderDepth = *(int*)GetSetMediaTrackInfo(track, "I_FOLDERDEPTH",
 			nullptr);
+		const string folderState = formatFolderState(track);
```

Comment:
nit: I think we can just call this in the two places we need it, rather than getting it here. This avoids pointless string generation for normal tracks.